R-package/.Rbuildignore

```diff
@@ -24,3 +28,20 @@ AUTOCONF_UBUNTU_VERSION
 ^src/compute/.gitignore$
 ^src/compute/CONTRIBUTING.md$
 ^src/compute/README.md$
+src/external_libs/fast_double_parser/benchmarks
+src/external_libs/fast_double_parser/Makefile
+src/external_libs/fast_double_parser/.*\.md
+src/external_libs/fast_double_parser/tests
```

Comment:
```suggestion
src/external_libs/fast_double_parser/tests
src/external_libs/fast_double_parser/.*\.yaml
src/external_libs/fast_double_parser/.*\.yml
```

To fix this issue from `R CMD CHECK`

```text
Found the following hidden files and directories:

  src/external_libs/fast_double_parser/.cirrus.yml
```

Reply:
@AlbertoEAF either you didn't accept this change, or you did and then force-pushed over it. But either way, can you please add it? Because right now it's the reason some of the R tests are failing.

By the way, you don't need to use `rebase`. We squash all commits on merge in this project, so you can just `merge`. Then you won't have to force-push.

Reply:
Must have been the rebase.